sample/src/main/java/kin/sdk/core/sample/ViewUtils.java

```diff
@@ -28,6 +33,32 @@ public static void alert(Context context, String message) {
         dialog.show();
     }
 
+    public static void confirmAlert(Context context, String message, String button, onAlertClicked onAlertClicked) {
```

Comment:
1. method name can be `alert`.
2. rename `String button` to `String confirmationMessage`
3. `OnConfirmListener confirmationListener`

Reply:
👍